### PR TITLE
Update Faq.jsx

### PR DESCRIPTION
**Pull request by 007AnupamSharma**
## Fixes Issue

This PR fixes the following issues:  [Enhancement] of FAQ component #614

closes #issue-number

## Changes proposed
fixed it ... in small devices, either one of those 6 needs to open, and after clicking on another accordion the previous one needs to be closed.

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes
image
both will open separately
image
 at a time only one faq component will open

### CODE REVIEW

**Comment by AnuragThePathak** [comment]
@007AnupamSharma fix the eslint errors.

**Comment by 007AnupamSharma** [comment]
@AnuragThePathak please check man I have made new commit

**Comment by AnuragThePathak** [comment]
@007AnupamSharma also fix the checklist thing. Take reference from previously merged PRs if needed.

**Comment by 007AnupamSharma** [comment]
@AnuragThePathak look i'm fresher in open-source .. so i'm not understanding what are u trying to say  .. please can you help me out

**Comment by AnuragThePathak** [comment]
@007AnupamSharma https://stackoverflow.com/questions/47344571/how-to-draw-checkbox-or-tick-mark-in-github-markdown-table

File: src/components/layout/Faq.jsx [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+    };
+  }, []);
+
   return (
     <section id='faq'>
       <div className='layout mx-auto py-20'>
         <h2 className='h1'>FAQs</h2>
         <hr className='styled-hr my-6' />
         <div className='grid-colos-1 grid gap-8 lg:grid-cols-2 lg:gap-6'>
-          <Accordion data={faq.slice(0, 3)}></Accordion>
```
Comment (Chandraprakash-Darji) [inline src/components/layout/Faq.jsx:10]: Make it one column by default, no two-column layout.

File: src/components/layout/Faq.jsx [hunk 2]
```diff
@@ -1,14 +1,35 @@
+// eslint-disable-next-line
 import Accordion from '@/components/Accordion';
-
+import { useState, useEffect } from 'react';
+// FAQ Component
 const Faq = ({ faq }) => {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
```
Comment (Chandraprakash-Darji) [inline src/components/layout/Faq.jsx:6]: As we are removing two column layout so no need for the state and effect

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: src/components/layout/Faq.jsx (hunk 1 above)
Comment (Chandraprakash-Darji) [reply src/components/layout/Faq.jsx:10]: Just loop over the faq and render the AccordionItem instead of Accordion, and remove the grid-cols-2 class

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Comment by 007AnupamSharma** [comment]
Like I had made the changes whatever you said @Chandraprakash-Darji please check it, bro

**Comment by AnuragThePathak** [comment]
@007AnupamSharma mark the conversations as resolved.

**Comment by 007AnupamSharma** [comment]
@AnuragThePathak Done

**Comment by Chandraprakash-Darji** [comment]
@007AnupamSharma I tried running locally, but it is not running?

**Comment by AnuragThePathak** [comment]
@007AnupamSharma are you still interested in working?

**Comment by AnuragThePathak** [comment]
Closing due to inactivity.

**Pull request closed without merging**